libbeat/tests/system/requirements.txt

```diff
@@ -29,12 +29,11 @@ ordered-set==3.1.1
 packaging==20.4
 parameterized==0.7.0
 pluggy==0.13.1
-py==1.10.0
+py==1.11.0
 pycodestyle==2.6.0
 pyparsing==2.4.7
 pyrsistent==0.16.0
-pytest==7.1.3
```

Comment:
Upgrade along with `py` to fix https://github.com/pytest-dev/apipkg/issues/30

I removed the `pytest-otel` dependency since it was pinning us to pytest 7.1.3 and wouldn't let me upgrade. I'm not sure it is strictly necessary anyway.

Apparently we own the `pytest-otel` package, I think we need to loosen the dependency to require `pytest >= 7.1.3` instead of `pytest == 7.1.3` if we can. FYI @kuisathaverat 

https://github.com/elastic/apm-pipeline-library/blob/cd076539fb75e75ca1a7fe740e5fac0abbf569f8/resources/scripts/pytest_otel/pyproject.toml#L5

```toml
dependencies = [
    "opentelemetry-api==1.15.0",
    "opentelemetry-exporter-otlp==1.15.0",
    "opentelemetry-sdk==1.15.0",
    "pytest==7.1.3",
]
```